.vscode/dryrun.log

```diff
@@ -0,0 +1,7 @@
+make.exe --dry-run --always-make --keep-going --print-directory
```

Comment:
Just making sure, do we need dryrun.log and targets.log or should they be in .gitignore?

Reply:
Thank you! We have added .vscode/*.log in .gitignore.